internal/worker_runner.go

```diff
@@ -232,7 +233,12 @@ func (wr *workerRunner) queryOBD() {
 				continue
 			}
 		} else if request.FormulaType() == models.Python {
-			value = hexResp[0] // todo: we need to experiment with real use case
+			// Convert to float
+			value, err = strconv.ParseFloat(hexResp[0], 64)
```

Comment:
in theory, the python formula could resolve to return a string, eg. parsing VIN out (although we don't use it for VIN) but imagine maybe like a Gear Positiion or some ADAS thing that could return a string from the python formula - which is why I think if the `hexResp` from the above function were changed to have more depth to it, this part would be better.